tests/test_data_loader.py

```diff
@@ -396,3 +398,15 @@ def test_end_of_dataloader_dispatcher(self):
         # Test it also works on the second iteration
         for idx, _ in enumerate(dataloader):
             assert dataloader.end_of_dataloader == (idx == 3)
+
+    def test_seedable_random_sampler_is_fully_applied(self):
+        dataloader = DataLoader(list(range(10)), batch_size=1, shuffle=True)
```

Comment:
Could you please instantiate the accelerator by calling `Accelerator()` at the top of the function body, same as in the other tests?

Reply:
Okay! I've add the initialization like `DataLoaderTester.test_end_of_dataloader_dispatcher` does. Should I pass `DataLoaderConfiguration` to `Accelerator` and call its `prepare()` method to do the test?